external/building-guides/building-archlinux-template.md

```diff
@@ -8,484 +8,437 @@ redirect_from:
 
-*   The VM should be based on a Fedora template. It's best to use a standalone VM. I created a standalone VM based on
-    the Fedora 23 template. I named the VM “**development**”. These instructions assume a standalone VM based on a       Fedora template is being used.
-<br>
-<br>
-![arch-template-01](/attachment/wiki/ArchlinuxTemplate/arch-template-01.png)
-<br>
-<br>
-*   Ensure there is at least 25GB preferably 30GB of free space in the private storage. I made the private storage 30GB to be safe.
-<br>
-<br>
 ![arch-template-02](/attachment/wiki/ArchlinuxTemplate/arch-template-02.png)
-<br>
-<br>
 
-*Note: Unless otherwise noted,  all commands are from within the “development” VM or whatever you named your standalone VM used for building the template.*
-
-## 2:   Create GitHub Account (optional)
 
+2:   Create GitHub Account (optional)
+-------------------------------------------
 *   It can be helpful. Creating only a basic account is all that is needed. This will allow you to help, going           forward, with the Qubes project. You could be help edit errors in documentation. It can also be of use building      other templates.
-
 *   Create user account here https://github.com
-<br>
-<br>
-![arch-template-03](/attachment/wiki/ArchlinuxTemplate/arch-template-03.png)
-<br>
-<br>
-
-## 3:   Install necessary packages to 'development' VM for "Qubes Automated Build System"
-
-*   Necessary packages to install:
-
-    *   git
-
-    *   createrepo
-
-    *   rpm-build
-
-    *   make
-
-    *   rpmdevtools
-
-    *   python3-sh
-
-    *   dialog
-
-    *   rpm-sign
-
-    *	gnupg
 
+![arch-template-03](/attachment/wiki/ArchlinuxTemplate/arch-template-03.png)
 
-*   The tools can usually be installed all together with the following terminal command string:
-
-    *   **$ sudo dnf install git createrepo rpm-build make wget rpmdevtools python3-sh dialog rpm-sign gnupg**
-<br>
-<br>
-![arch-template-04](/attachment/wiki/ArchlinuxTemplate/arch-template-04.png)
-<br>
-<br>
-
-## 4: Downloading and verifying the integrity of the "Qubes Automated Build System"
+3:   Install necessary packages to 'build-archlinux2' qube for "Qubes Automated Build System"
+-----------------------------------------------------------------------------------------------
+```shell_session
+# dnf install git make 
+```
 
+4: Downloading and verifying the integrity of the "Qubes Automated Build System"
+---------------------------------------------------------------------------------
 * Import the Qubes master key
-
-      gpg --keyserver pgp.mit.edu --recv-keys 0xDDFA1A3E36879494
-
+```shell_session
+$ gpg --import /usr/share/qubes/qubes-master-key.asc
+```
 * Verify its fingerprint, set as 'trusted'. [This is described here](/doc/VerifyingSignatures).
-
-* Download the Qubes developers' keys.
-
-      wget https://keys.qubes-os.org/keys/qubes-developers-keys.asc
-      gpg --import qubes-developers-keys.asc
```

Comment:
This step is necessary, just master key isn't enough.

Reply:
Indeed, thanks